Dockerfile

```diff
@@ -1,67 +1,17 @@
-FROM registry.access.redhat.com/ubi8/nodejs-20:latest AS BUILD_IMAGE
-
-### BEGIN REMOTE SOURCE
```

Comment:
@lavocatt , We need all these changes added within the BEGIN to END remote source. because i added those lines to keep sync with the midstream and downstream repos. For now, could you please revert these changes?